org.eclipse.m2e.editor.lemminx/pom.xml

```diff
@@ -60,7 +60,13 @@
 				<groupId>org.eclipse.tycho</groupId>
 				<artifactId>tycho-packaging-plugin</artifactId>
 				<configuration>
-					<jgit.ignore>nothing</jgit.ignore> <!-- empty (whitespace needed), don't ignore pom.xml as changes in here affect payload -->
+					<!-- overrides from parent as we don't want to ignore pom changes! -->
+					<ignores>
+						<ignore>OSGI-INF/*</ignore>
+						<ignore>META-INF/ECLIPSE_.RSA</ignore>
+						<ignore>META-INF/ECLIPSE_.SF</ignore>
```

Comment:
```suggestion
						<ignore>META-INF/ECLIPSE_.*</ignore>
```

Reply:
I think I will rvert this alltogehter and dont touch it.